tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1334,30 +1334,22 @@ public void NativeAOTSample ()
 		}
 
 		[Test]
-		public void EnableManagedMarshalMethodsLookup ()
+		public void AppStartsWithManagedMarshalMethodsLookupEnabled ()
```

Comment:
Is the old system being completely removed? Do we not need the old test?

Reply:
This test was introduced in this PR. I just changed the name of the test in the last commit.